src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -439,20 +530,11 @@ class KubeflowTrainingService implements TrainingService {
                         name: 'tensorflow',
                         image: replicaImage,
                         args: ["sh", `${path.join(trialWorkingFolder, runScriptFile)}`],
-                        volumeMounts: [{
-                            name: 'nni-nfs-vol',
-                            mountPath: this.CONTAINER_MOUNT_PATH
-                        }],
+                        volumeMounts: this.kubeflowClusterConfig.nfs?volumeSpecMap.get('nfsVolumeMounts'):volumeSpecMap.get('azureVolumeMounts'),
```

Comment:
If 'nfsVolumeMounts' and 'azureVolumeMounts' are merged into one key, like 'nniVolumeMounts', you can avoid conditional operator, just all volumeSpecMap.get('nniVolumeMounts') is enough